CodePush.js

```diff
@@ -431,12 +431,18 @@ function codePushify(options = {}) {
 
           let syncStatusCallback;
           if (rootComponentInstance && rootComponentInstance.codePushStatusDidChange) {
-            syncStatusCallback = rootComponentInstance.codePushStatusDidChange.bind(rootComponentInstance);
+            syncStatusCallback = rootComponentInstance.codePushStatusDidChange;
+            if (rootComponentInstance instanceof React.Component) {
```

Comment:
Shouldn't we still bind `this` to the object that was passed into `createClass()`? Or is it just too difficult to get access to that object?